pyproject.toml

```diff
@@ -63,24 +54,14 @@ build = [
     'Cython>=0.29.32',
     'pythran',
     'numpy>=1.22',
+    # Developer UI
     'spin==0.4',
     'build',
 ]
 data = ['pooch>=1.6.0']
-default = [
```

Comment:
> [...] But in pyproject.toml it is duplicated. From pip's perspective pip install scikit-image and pip install scikit-image[default] do the same thing. [...] -- https://github.com/scikit-image/scikit-image/pull/7085#issuecomment-1666142890

Ah, got you, nice catch. Happy to remove it. :)

Reply:
If, by chance, anyone tries `pip install --dry-run scikit-image[default]` they would see "WARNING: scikit-image 0.22.0rc0 does not provide the extra 'default'". But the installed set of packages is the same.

Reply:
Perfect. :)